litex/tools/litex_json2dts_zephyr.py

```diff
@@ -137,10 +166,28 @@ def generate_dts_config(csr):
         if name not in overlay_handlers.keys():
             print('No overlay handler for:', name, 'at', hex(value))
 
+    cnf += ' -DCONFIG_LITEX_CSR_DATA_WIDTH={}'.format(
+        csr['constants']['config_csr_data_width'],
+    )
+
     return dts, cnf
 
 
 # helpers
+def get_registers_of(name, csr):
+    registers = csr['csr_registers']
+    data_width = csr['constants']['config_csr_data_width']
+
+    return [
+        {
+            **params,
+            'size': params['size'] * 4,
```

Comment:
a comment why we are multiplying `size` would be great

Reply:
Done